src/main/java/org/openrewrite/javascript/tree/JS.java

```diff
@@ -1678,7 +1678,8 @@ public CoordinateBuilder.Expression getCoordinates() {
         }
 
         public enum Type {
-            ReadOnly
+            ReadOnly,
+            Keyof,
```

Comment:
This makes it in line with `KeyOfKeyword`.
```suggestion
            KeyOf,
```